lib/src/main/java/co/nimblehq/alpine/lib/nfc/ImageUtil.java

```diff
@@ -0,0 +1,141 @@
+package co.nimblehq.alpine.lib.nfc;
+
+import android.content.Context;
+import android.graphics.*;
+
+import org.jmrtd.lds.AbstractImageInfo;
+import org.jnbis.WsqDecoder;
+
+import java.io.*;
+
+import jj2000.j2k.decoder.Decoder;
+import jj2000.j2k.util.ParameterList;
+
+/*
+ * Copyright 2016 Anton Tananaev (anton.tananaev@gmail.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+public class ImageUtil {
```

Comment:
I'm not sure shall we transform this class to kotlin too? 🤔

Reply:
@lydiasama how about I create a new chore ticket and assign it to you for this? 😆

Reply:
@AVI5HEK It's better to separate it into a chore one. 👍🏻